app/Models/Organization.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Interns2025b\Models;
+
+use Carbon\Carbon;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+
+/**
+ * @property int $id
+ * @property string $name
+ * @property ?string $group_url
+ * @property ?string $avatar_url
+ * @property ?int $owner_id
+ * @property Carbon $created_at
+ * @property Carbon $updated_at
+ */
+class Organization extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        "name",
+        "group_url",
+        "avatar_url",
+        "owner_id",
+    ];
+    protected $casts = [
+        "owner_id" => "integer",
```

Comment:
You may want to reconsider whether you want to use uuid in all models instead of integer